.github/workflows/release-upload.yaml

```diff
@@ -179,15 +180,28 @@ jobs:
           # TODO generalize
           # tar -czf ${{ needs.wait_for_hydra.outputs.TARGET_TAG }}-macos.tar.gz cardano-cli-macos
           # zip ${{ needs.wait_for_hydra.outputs.TARGET_TAG }}-win64.zip cardano-cli-win64
+      - name: Create short tag
+        run: |
+          # Transform the long tag (e.g. "cardano-cli-8.22.0.0")
+          # into the short version (e.g. "8.22.0.0")
+          long_tag=${{ needs.wait_for_hydra.outputs.TARGET_TAG }}
+          short_tag="${long_tag#cardano-cli-}"
+          echo "SHORT_TAG=$short_tag" >> "$GITHUB_ENV"
+      - name: Create changelog
+        run: |
+          echo -e "# Changelog\n" > RELEASE_CHANGELOG.md
+          ./scripts/ci/extract-changelog.sh ${{ env.SHORT_TAG }} >> RELEASE_CHANGELOG.md
       - name: Create Release
         uses: input-output-hk/action-gh-release@v1
         if: ${{ needs.wait_for_hydra.outputs.DRY_RUN == 'false' }}
         with:
           draft: true
-          tag_name: ${{ needs.wait_for_hydra.outputs.TARGET_TAG }}
+          tag_name: ${{ needs.wait_for_hydra.outputs.TARGET_TAG }} # Git tag the release is attached to
+          name: ${{ env.SHORT_TAG }} # Release name in GitHub UI
           # TODO generalize
           # cardano-cli-${{ needs.wait_for_hydra.outputs.TARGET_TAG }}-macos.tar.gz
```

Comment:
https://github.com/IntersectMBO/cardano-cli/pull/686 got merged, so I think we can enable those too, right?

Reply:
@carbolymer> yep I'm working on it, but it will be a separate PR :+1:

Reply:
WIP PR here for support of Darwin: https://github.com/IntersectMBO/cardano-cli/pull/735